### DISCUSSION TITLE
Cannot write to MATRIX room and XMPP MUC missing

### DISCUSSION THREAD

**Discussion by roughnecks** [TriliumNext/Notes#289] [General]
Hello,
just found out about TriliumNext and I joined the MATRIX room posted in the README. When I tried to greet, this happened:

`Failed to send message: M_FORBIDDEN: You don't have permission to post that to the room.`
`user_level (0) < send_level (100)`

And you're also mentioning XMPP but cannot find any MUC JID.

Thanks

**Comment by roughnecks** [comment]
Nevermind, the MATRIX room is a space and I was able to join support now.
XMPP still missing tho :)

**Reply by meichthys** [reply]
I think the XMPP link is in the matrix space description. If not, let me know and I can update it.

**Reply by roughnecks** [reply]
yeah, just joined. Would be good to also have it on github readme, I think. Thanks
